user-onboarding/src/components/form.js

```diff
@@ -0,0 +1,130 @@
+import React from 'react'
+
+export default function Form(props) {
+   const { submit, errors, disabled, values, change } = props
+
+   const onSubmit = event => {
+      event.preventDefault()
```

Comment:
If you moved the event.preventDefault() into the submit function in app.js you wouldn't need to declare this function and could call submit right on the form action